projects/fsastorefrontlib/src/cms-components/my-account/policy/policy.module.ts

```diff
@@ -60,9 +64,15 @@ const routes: Routes = [
     ChangeRequestStoreModule,
     MediaModule,
     DocumentsTableModule,
+    NgxEchartsModule.forRoot({
```

Comment:
Since this would always load the charts when starting an app, I would suggest implementing lazy loading for charts.
More on the subject can be found here:
https://medium.com/@tomastrajan/why-and-how-to-lazy-load-angular-libraries-a3bf1489fe24

Reply:
As we agreed I'll add task for improvement for this.